Snowflake.Data.Tests/IntegrationTests/SFConnectionPoolIT.cs

```diff
@@ -395,5 +396,31 @@ public void TestConnectionPoolTurnOff()
             SnowflakeDbConnectionPool.SetPooling(false);
             //Put a breakpoint at SFSession close function, after connection pool is off, it will send close session request.
         }
+
+        [Test]
+        public void TestCloseSessionAfterTimeout()
+        {
+            // arrange
+            const int SessionTimeoutSeconds = 2;
+            const int TimeForBackgroundSessionCloseMillis = 2000;
+            SnowflakeDbConnectionPool.SetTimeout(SessionTimeoutSeconds);
+            var conn1 = new SnowflakeDbConnection(ConnectionString);
+            conn1.Open();
+            var session = conn1.SfSession;
+            conn1.Close();
+            Assert.IsTrue(session.IsEstablished());
+            Thread.Sleep(SessionTimeoutSeconds * 1000);
+            var conn2 = new SnowflakeDbConnection(ConnectionString);
```

Comment:
I had to wonder a bit why all these steps are needed. Maybe it's worth adding comments, eg. 
- wait for session timeout
- wait for cleaning expired session

Reply:
I'm a litle bit reluctant to do so because the names of the const values are already expressing it.
But I added such comments.